dbms/src/Storages/FormatVersion.h

```diff
@@ -130,6 +131,15 @@ inline static const StorageFormatVersion STORAGE_FORMAT_V5 = StorageFormatVersio
     .identifier = 5,
 };
 
+inline static const StorageFormatVersion STORAGE_FORMAT_V6 = StorageFormatVersion{
+    .segment = SegmentFormat::V2,
+    .dm_file = DMFileFormat::V3,
+    .stable = StableFormat::V2, // diff
+    .delta = DeltaFormat::V3,
+    .page = PageFormat::V3,
+    .identifier = 6,
+};
+
 // STORAGE_FORMAT_V100 is used for S3 only
 inline static const StorageFormatVersion STORAGE_FORMAT_V100 = StorageFormatVersion{
```

Comment:
Do we need a V101? Is V100 used? cc @JaySon-Huang

Reply:
When it is deployed in disaggregated, it will use `STORAGE_FORMAT_V100`. And yes we need a V101 for disaggregated mode.

https://github.com/pingcap/tiflash/blob/961f9dded38b814bb41b33c691ab58f3f090a0d9/dbms/src/Server/Server.cpp#L975-L1001